source/_components/device_tracker.owntracks_http.markdown

```diff
@@ -47,17 +47,6 @@ sensor:
         friendly_name: Pixel of Paulus
         unit_of_measurement: "%"
         value_template: '{{ states.device_tracker.paulus_pixel.attributes.battery|int }}'
-        icon_template: >-
```

Comment:
I removed this nice template here because the `device_class` does the dynamic battery level icon. Still, this template is quite a nice way to learn more about the template engine, at least I myself learned a lot about how jinja2 worked using this template.